### PR TITLE
feat: Highlight email box in sign in and sign up forms and display message if email invalid

### LINKED ISSUE #252: [FEATURE] Highlight email input if its wrong.
### Description

Highlight email input if its invalid (e.g. name@example. ). Also display a message below email input.
(very simple but I would be happy to contribute :) )

### PR DESCRIPTION
**Pull request by sohamdata**
### 🛠️ Fixes Issue

Closes #252 

### ➕ Changes Introduced

* The email box ring and text is highlighted in tailwind color pink-600. "Please provide a valid email address" is displayed under it.
* Minor change: Changed Text color of "Join us on discord, twitch" to match more to their respective logos.

### 📷 Screenshots

Attached in the issue #252

### CODE REVIEW

File: src/components/CallToAction.jsx [hunk 1]
```diff
@@ -23,10 +23,10 @@ export function CallToAction() {
           </p>
           <div className='mt-10 flex justify-center gap-x-6'>
             <Button href='https://discord.com/invite/nNtVfKddDD' target='_blank' color='white'>
-              <span className='text-blue-600'>Join us on Discord</span>
+              <span className='text-[#7289DA]'>Join us on Discord</span>
             </Button>
             <Button href='https://www.twitch.tv/ykdojo' target='_blank' color='white'>
-              <span className='text-purple-600'>Join us on Twitch</span>
+              <span className='text-[#9146FF]'>Join us on Twitch</span>
```
Comment (ykdojo) [inline src/components/CallToAction.jsx:29]: If you want to make this change, can you send this as a separate PR?

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: src/components/CallToAction.jsx (hunk 1 above)
Comment (sohamdata) [reply src/components/CallToAction.jsx:29]: You can exclude it, it's fine. It's too minor of a change to notice anyways.

File: src/components/CallToAction.jsx [hunk 2]
```diff
@@ -23,10 +23,10 @@ export function CallToAction() {
           </p>
           <div className='mt-10 flex justify-center gap-x-6'>
             <Button href='https://discord.com/invite/nNtVfKddDD' target='_blank' color='white'>
-              <span className='text-blue-600'>Join us on Discord</span>
+              <span className='text-[#7289DA]'>Join us on Discord</span>
```
Comment (ykdojo) [inline src/components/CallToAction.jsx:26]: can you send this one as a separate PR?

**Review by ykdojo — CHANGES_REQUESTED** [review]

File: src/components/CallToAction.jsx (hunk 2 above)
Comment (sohamdata) [reply src/components/CallToAction.jsx:26]: ya sure.

**Comment by ykdojo** [comment]
Just checked the preview and it shows this error message even when there's nothing entered yet.

I think we should fix that:

image

**Comment by sohamdata** [comment]
I'll see what i can do.
